tools/nightly-builder/latest-build.js

```diff
@@ -0,0 +1,36 @@
+"use strict"
+
+const jsonist = require('jsonist')
+
+const indexUrl = 'https://iojs.org/download/{type}/index.json'
+
+
+function latestBuild (type, callback) {
+  let url = indexUrl.replace(/\{type\}/, type)
```

Comment:
well the idea here is that I want to keep these kinds of fixed strings out of the logic body, template strings don't really help with that because you have to piece multiple bits together